app/models/concerns/publishable.rb

```diff
@@ -14,27 +14,25 @@ module Publishable
 
   def publish!
     publish_content_item!
-    publish_rummager_artefact!
+    add_to_search_index!
+  end
+
+  def base_path
+    "/government/policies/#{slug}"
```

Comment:
Where possible it would be good not to reference policy directly in this concern so we can use it for other things (like email signup pages).